WordPress/twentytwentyfive
# Add typography presets

**Description**
This PR adds the typography presets.
Partial for https://github.com/WordPress/twentytwentyfive/issues/22

**Screenshots**

**Testing Instructions**
Apply the PR.
Go to Appearance > Editor > Styles
Test every typography preset in the editor, save, and view the preset on the front.
Confirm if the typography matches the design.


- [x] `typography-preset-1` Beiruti & Literata [Figma](https://www.figma.com/design/dzGCSntVch4EQdVERTqyVK/Twenty-Twenty-Five?node-id=77-581&t=yraxs70Mo4pEIrsW-4)
- [x] `typography-preset-2` Volkorn & Fira Code [Figma](https://www.figma.com/design/dzGCSntVch4EQdVERTqyVK/Twenty-Twenty-Five?node-id=77-338&t=Q7SIN1k9o8CVCm5L-0)
- [x]  `typography-preset-3` Platypi & Ysabeau Office [Figma](https://www.figma.com/design/dzGCSntVch4EQdVERTqyVK/Twenty-Twenty-Five?node-id=77-375&t=Q7SIN1k9o8CVCm5L-4)
- [x]  `typography-preset-4` Roboto Slab & Manrope [Figma](https://www.figma.com/design/dzGCSntVch4EQdVERTqyVK/Twenty-Twenty-Five?node-id=77-412&t=Q7SIN1k9o8CVCm5L-4)
- [x]  `typography-preset-5` Literata & Ysabeau Office [Figma](https://www.figma.com/design/dzGCSntVch4EQdVERTqyVK/Twenty-Twenty-Five?node-id=77-617&t=Q7SIN1k9o8CVCm5L-4)
- [x]  `typography-preset-6` Platypi & Literata [Figma](https://www.figma.com/design/dzGCSntVch4EQdVERTqyVK/Twenty-Twenty-Five?node-id=77-680&t=Q7SIN1k9o8CVCm5L-4)
- [x]  `typography-preset-7` Literata & Fira Sans [Figma](https://www.figma.com/design/dzGCSntVch4EQdVERTqyVK/Twenty-Twenty-Five?node-id=77-476&t=Q7SIN1k9o8CVCm5L-4)


https://github.com/user-attachments/assets/148bc21e-54a5-499f-a6f9-806602493d7d
